java/code/src/com/redhat/rhn/frontend/xmlrpc/contentmgmt/ContentManagementHandler.java

```diff
@@ -529,27 +529,31 @@ public ContentFilter lookupFilter(User loggedInUser, Integer id) {
      * @xmlrpc.doc Create a Content Filter
      * #paragraph_end()
      * #paragraph()
-     * The following filters are available:
+     * The following filters are available (you can get the list in machine-readable format using
+     * the listFilterCriteria() endpoint):
      * #paragraph_end()
      * #paragraph()
      * Package filtering:
      * #itemlist()
-     *   #item("by name - field:name matcher:contains")
-     *   #item("by name, epoch, version, release and architecture - field:nevr or nevra - matcher:equals")
+     *   #item("by name - field:name; matchers:contains or matches")
+     *   #item("by name, epoch, version, release and architecture - field:nevr or nevra; matcher:equals")
      *  #itemlist_end()
      * #paragraph_end()
      * #paragraph()
      * Errata/Patch filtering:
      * #itemlist()
-     *   #item("by advisory name - field:advisory_name matcher:equals")
-     *   #item("by synopsis - field:synopsis matcher:equals or contains")
-     *   #item("by keyword - field:keyword matcher:contains")
-     *   #item("by date - field:issue_date matcher:greater or greatereq")
-     *   #item("by type - field:advisory_type matcher:equals")
-     *   #item("by affected package name - field:package_name matcher:contains_pkg_name")
-     *   #item("by affected package with version - field:package_nevr matcher:contains_pkg_lt_evr, contains_pkg_le_evr,
+     *   #item("by advisory name - field:advisory_name (e.g. 'Security Advisory'); matcher:equals or matches")
```

Comment:
I think "advisory_name" is not "Security Advisory" but more "SUSE-2019-1234".

advisory_type can be a "Security Advisory"

Reply:
Good catch!